packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -169,67 +169,73 @@ def _log_arguments(
     return extra_args
 
 
+P = ParamSpec("P")
+R = TypeVar("R")
+
+
+@overload
+def log_decorator(
+    logger: logging.Logger | None = None, level: int = logging.DEBUG
+) -> Callable[[Callable[P, R]], Callable[P, R]]:
+    ...
+
+
+@overload
 def log_decorator(
-    logger=None, level: int = logging.DEBUG, *, log_traceback: bool = False
```

Comment:
thx for removing the "log_traceback" ... eeeeendlesss discussions about thisone!  Perhaps we could expose instead the arguments of the logger ...
Moreover, I personally think we could move the design of `pytest_simcore.helper.logging_tools` here https://github.com/itisfoundation/osparc-simcore/blob/de48da0a2319e34de8b31485fafa94d8ebaff651/packages/pytest-simcore/src/pytest_simcore/helpers/logging_tools.py#L114-L121 here

Reply:
yep I agree.